backend/services/inventory/api/http/api_inventory.go

```diff
@@ -529,8 +530,18 @@ func (i *InternalAPI) PatchDeviceAttributesInternalHandler(
 		}
 		notModifiedAfter = &parsed
 	}
+	scopeFromUrl := true
+	r := strings.NewReplacer(":tenant_id", tenantId, ":device_id", deviceId)
+	if strings.Contains(
+		c.Request.RequestURI,
+		r.Replace(urlInternalAttributesNoScope),
+	) {
+		scopeFromUrl = false
+	}
```

Comment:
This could be simplified to a simpler expresion:
```suggestion
	scopeFromUrl := c.FullPath() == (apiUrlInternalV1 + urlInternalAttributes)
```

Reply:
amazin! I really forgot about the FullPath existence. changed.